packages/actions/src/Imports/ImportColumn.php

```diff
@@ -437,6 +452,14 @@ public function getExample(): mixed
         return $this->evaluate($this->example);
     }
 
+    /**
+     * @return array<mixed>
+     */
+    public function getExamples(): array
+    {
+        return array_merge([$this->evaluate($this->example)], $this->examples);
+    }
```

Comment:
First, call `$this->evaluate($this->examples)`, then `Arr::wrap()` that, then return it